db/models/articles-model.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../connection");
+
+const readArticleByID = (article_id) => {
+  if (isNaN(+article_id)) {
+    return Promise.reject({ status: 400, msg: "bad request" });
+  }
```

Comment:
We don't need to handle the 400 error here, a psql error will automatically be thrown if the ID is not a number 👍